templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url "taxi:driver-detail" pk=user.id %}">User: {{ user.username }}</a></li>
+    <li><a href="{% url "taxi:logout" %}">Logout</a></li>
+  {% else %}
+  <li><a href="{% url "taxi:login" %}">Login</a></li>
```

Comment:
The URL name `taxi:login` may not be correct if you are using Django's built-in authentication URLs. Typically, the login URL is `login` without the app namespace. Ensure that the URL name matches the one defined in your URL configuration.